exercises/series/src/example/java/Series.java

```diff
@@ -4,21 +4,20 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class Series {
+class Series {
 
     private final int digitsSize;
     private List<Integer> digits;
 
-    public Series(String string) {
-        this.digits = Arrays
-                .asList(string.split(("")))
-                .stream()
-                .map(digit -> Integer.parseInt(digit))
-                .collect(Collectors.toList());
+    Series(String string) {
+        this.digits =
+                Arrays.stream(string.split(("")))
```

Comment:
@FridaTveit 
thanks for your suggestions, I'll update the PR as soon as get back home.

Is there any industry accepted best practices for code formatting (as you mentioned) or it's usually specific to the project (if so, please point me to the page describing this rules)?